### DISCUSSION TITLE
Missing watchtower insights

### DISCUSSION THREAD

**Discussion by Zetanova** [lightningnetwork/lnd#10263] [Ideas]
Currently there are now rpc method to query the watchtower 
for information what is stored inside the watchtower.db

Info's client about stored channel-data and backups would be 
"nice to have" and maybe to delete/manage/scavenge them.

**Comment by Chinwendu20** [comment]
Hello all please assign this issue to me.  I would be working on adding these insights:
- How many sessions has the tower stored
- How many of these sessions are active, exhausted, feel free to suggest any other.

I would be doing this by adding the stats attribute to the server just like it is in the client.

**Comment by anibilthare** [comment]
@guggero can you please suggest what all insights might be good to have ?

**Comment by guggero** [comment]
>  - How many sessions has the tower stored
>  - How many of these sessions are active, exhausted, feel free to suggest any other.


I think addressing the items listed above would be a good start. Once you're familiar with the code and have tested things out locally, I'm sure you'll notice other information that could be useful to be exposed to the operator of a watchtower server.
